### ISSUE TITLE
announcing of leaving and joining people by a screen reader.

### ISSUE THREAD

**Issue by audioses** [bigbluebutton/bigbluebutton#11448]
Hello,
i love the accessibility of the big blue button, but i have a suggestion.
currently, the screen readers are not announcing when someone leaves/joins a meeting.
is it possible to improve that feature so the screen readers can read the other notifications?
join/leaving
new slide publishing etc etc
The screen reader only announces when someone shares or stops sharing the screen and when someone sends a message.
It would be nice if the functionalety of this  feature can be improved.

**Comment by antobinary** [comment]
Hi @audioses 

I was looking through the remaining open 'accessibility' issues...

Is this still an issue if you've enabled the notifications in Settings? I am attaching a screenshot which displays the Settings in BigBlueButton 2.4.0 (has not changed much in structure or content since 2.3.0 from what I can tell but recently was part of an accessibility audit and did well).
In the Settings there is a Notifications tab including toggles for audio alerts and popup alerts (toasts) for each of the following:
Chat message
User join
User leave
Raise hand

The default values are to not alert for anything but raise hand.

Could you (or @KDSBrowne ) confirm whether enabling alerts for user join/leave is sufficient to address this issue?

![Screenshot from 2022-01-18 08-50-26](https://user-images.githubusercontent.com/6312397/149950106-a675fa5f-f896-4394-8592-c77893820418.png)

**Comment by KDSBrowne** [comment]
@antobinary The screen reader does announce User Join / Leave  when the popup alert is enabled in the settings.

**Comment by audioses** [comment]
Hello there,
  I have tested this by using demo.bigbluebutton.org and  it seems to be working now.
  I never checked the settings so never knew that option existed. Thanks for the improvements 👍 )

**Comment by antobinary** [comment]
I extracted the missing portion into a new issue #14141 so we can close this one. Thank you both!
